### PR TITLE
fix:remove moonfit campaign

### PR DESCRIPTION
**Pull request by gluneau**
**Pull Request Summary**

- remove moonfit campaign
- add Algem logo

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

- (ex: Add feature A)

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

> \*Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Would be good to include this but Algem lost the logo from the banner. could you take a look?

**Comment by Kahonnohak** [comment]
Hi we don't need a new logo, can we have the logo back that we had before - you can see in other PR e.g) https://astar-apps--pr974-feat-bridge-erc20-6jsmcjx5.web.app/astar/dapp-staking/discover
Screenshot 2023-10-23 at 20 46 14

File: src/components/dapp-staking/my-staking/DynamicAdsArea.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default defineComponent({
         } else {
           return require('/src/assets/img/ic_astar_farm.svg');
         }
-      } else if (index === 2) {
+      } else if (index === 1) {
```
Comment (gluneau) [inline src/components/dapp-staking/my-staking/DynamicAdsArea.vue:93]: This is overriding code for the first two campaigns.  Their "img" can remain empty.

**Review by bobo-k2 — APPROVED** [review]

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Pull request merged**